### ISSUE TITLE
resource gathering working in bad spots

### ISSUE THREAD

**Issue by UltraBlackLinux** [Exortions/skyblock-2.0#31]
Some places like the house in the forest are in an area that's used for gathering resources. In this case one is able to mine logs. The house however is also made of logs and you are thus able to mine it
![screenshot-20221219-001613](https://user-images.githubusercontent.com/62404294/208324543-6e7227bc-a713-4ae0-8043-331854ad1956.png)

**Comment by Exortions** [comment]
Yeah, it's linked to your Location, and we would have to re-create that location, it's kind of annoying to do

**Comment by Exortions** [comment]
Shouldn't be a big deal though, as the wood re-generates

**Comment by UltraBlackLinux** [comment]
it does but that does not work on hypixel. Just a minor nitpick.

**Comment by Exortions** [comment]
yeah, will fix later though.

**Comment by Exortions** [comment]
yeah, will fix later though.
